src/letsRoute.js

```diff
@@ -43,7 +43,24 @@ Anumargak.prototype.on = function (method, url, fn) {
 var wildcardRegexStr = "\\/([^\\/:]*)\\*";
 var paramRegexStr = ":([^\\/\\-\\(]+)-?(\\(.*?\\))?";
 var enumRegexStr = ":([^\\/\\-\\(]+)-?(\\(([\\w\\|]+)\\))";
-var namedExpressionRegexStr = "\\(:(.*?):\\)";
+
+function replaceNamedExpression(url, _this) {
```

Comment:
You may change it to `Anumargak.prototype.replaceNamedExpression()`. So you'll not need to pass `this`.